pingouin/external/tabulate.py

```diff
@@ -5,7 +5,8 @@
 
 from __future__ import print_function
 from __future__ import unicode_literals
-from collections import namedtuple, Iterable
+from collections import namedtuple
+from collections.abc import Iterable
```

Comment:
Importing `Iterable` from `collections` will stop working in 3.8. It was raising a deprecation warning.

Reply:
Great thanks!